_sass/libs/rivet-uits/utilities/_utilities-flex.scss

```diff
@@ -1,8 +1,5 @@
 // rivet-uits - @version 1.3.0
 
-// Copyright (C) 2018 The Trustees of Indiana University
```

Comment:
Just checking if this was removed on purpose?

Reply:
Oh. No. But that's going to keep happening each time we update because it's being pulled into the project from Rivet. I ran the update to get Rivet's JS.

[I made a ticket in the Rivet source repo](https://github.com/indiana-university/rivet-source/issues/90)

Reply:
Also pushed an update to add the license back on our end.